src/utils/dmx/BasicColor.js

```diff
@@ -7,7 +7,8 @@ export default class BasicColor extends DmxDevice {
   constructor(options) {
     super(Object.assign({}, options, {
       params: {
-        color: new RgbParam([1, 2, 3])
+        color: new RgbParam([1, 2, 3]),
+        dimmer: new RangeParam(1, { min: 0, max: 255 })
```

Comment:
I'll be honest, I probably didn't understand entirely what I was doing with this when I wrote it. I'll take a look at this evening and get back to you with an explanation or changes.

Reply:
I'm pretty sure I needed a way for basic 3-color RGB lights to be controlled with a dimmer parameter, since that's how the shake, strobe, and fade animations operate. I don't think this is the best solution, so I'll still be taking a closer look at it tonight.

Reply:
Okay, I've revised this so it doesn't modify the existing BasicColor fixture - it adds a new fixture which implements this functionality. Also, its value is overridden by the red channel. I think this is because of the order in which the properties are processed. In either case, it seems to be working in my testing, and at very least it won't break any existing functionality you have.

Sorry I wasn't more careful or considerate about that kind of thing. I'll try to be more careful in the future.